tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -294,11 +296,12 @@ class AdaptiveQueryExecSuite
   }
 
   test("Avoid transitions to row when writing to Parquet") {
-
+    logError("Avoid transitions to row when writing to Parquet")
```

Comment:
Are these logs leftover from a debug session?  Normally we don't log errors in the tests.

Reply:
Yes, but we have no good way to tell what logs came from which test. Normally we have not been doing it, but debugging without something like it is really painful.